core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -2178,6 +2178,20 @@ protected function buildManagerImagePreview($path, $ext, $width, $height, $bases
             } catch (Exception $E) {
                 $timestamp = 0;
             }
+
+            if ($bases['pathIsRelative']) {
+                $absolute_path = $bases['pathAbsolute'] . $path;
+            } else {
+                $absolute_path = $path;
+            }
+            $exif = @exif_read_data($absolute_path);
```

Comment:
What do you think about adding a `function_exists('exif_read_data')`?

Reply:
Yes, right, thank you =))

Reply:
Thank you too!